flash_examples/predict/summarization.py

```diff
@@ -48,7 +48,7 @@
 print(predictions)
 
 # 2b. Or generate summaries from a sheet file!
-datamodule = SummarizationData.from_file(
```

Comment:
is there any warning so that the users can notice the change of this API  ?

Reply:
The API is still supported.